nested/db-mssql.json

```diff
@@ -48,7 +48,7 @@
                     "type": "firewallRules"
                 },
                 {
-                    "apiVersion": "2014-04-01",
+                    "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.